packages/world/mud.config.ts

```diff
@@ -48,6 +48,21 @@ export default defineWorld({
       },
       key: ["objectTypeId"],
     },
+    ObjectTypeOreAmount: {
+      schema: {
+        objectTypeId: "ObjectTypeId",
+        oreTypeId: "ObjectTypeId",
+        amount: "uint16",
```

Comment:
Max uint16 is 65,535, maybe should be bigger?

I think there'll be more than these in the world, or can add a todo and change later once we have the exact count from the map

Reply:
this is just to track how many ores should be burned for that objectTypeId

Reply:
ahh right